bin/monitor_queue_sizes

```diff
@@ -0,0 +1,8 @@
+#/usr/bin/env bash
+
+# Once a minute, emit the number of jobs in each queue to Librato
+
+while true; do
+  bundle exec rake monitor:queue_sizes
+  sleep 60
```

Comment:
Should this an env var so that it could be overridden, e.g. `sleep ${RESQE_BRAIN_MONITOR_QUEUE_SIZE_TIMEOUT-60}`?  Or is that YAGNI?